region/norfair/crocomire/Grapple Beam Room.json

```diff
@@ -161,27 +183,64 @@
       "requires": []
     },
     {
-      "link": [3, 1],
+      "link": [2, 4],
       "name": "Base",
       "requires": [
         {"or": [
-          {"and": [
-            "HiJump",
-            {"or": [
-              "Grapple",
-              "canCarefulJump"
-            ]}
-          ]},
-          "canSpringBallJumpMidAir",
+          "HiJump",
           "canWalljump",
-          "h_canFly"
+          "h_canCrouchJumpDownGrab",
+          "canSpringBallJumpMidAir"
         ]}
       ]
     },
     {
       "link": [3, 2],
       "name": "Base",
       "requires": []
+    },
+    {
+      "link": [4, 1],
+      "name": "Base",
+      "requires": [
+        {"or": [
+          "Grapple",
+          "canPreciseWalljump"
+        ]}
+      ]
+    },
+    {
+      "link": [4, 1],
+      "name": "HiJump",
+      "requires": [
+        "HiJump",
+        "canCarefulJump"
```

Comment:
Speed also removes the careful jump, but probably not worth including on basic, since it's obscure enough?